foundations/html_css/css-foundations/intro-to-css.md

```diff
@@ -13,8 +13,6 @@ This section contains a general overview of topics that you will learn in this l
 For a more interactive explanation and example, try the following Scrim (let us know what you think of these):
 
 <iframe src="https://scrimba.com/scrim/co12d4cf99cf2776f19e84a9d?embed=odin,mini-header,no-sidebar,no-next-up" sandbox="allow-scripts allow-same-origin allow-popups" width="100%" height="400"></iframe>
-
-
 ### Basic Syntax
```

Comment:
there's an empty line needed above this header. Please make sure to check all sections that changed with our [preview tool](https://www.theodinproject.com/lessons/preview)